src/status_im/contexts/chat/messenger/messages/list/style.cljs

```diff
@@ -25,12 +24,10 @@
   [bottom theme top-margin]
   (reanimated/apply-animations-to-style
    {:bottom bottom}
-   (merge
-    (shadows/get 2 (quo.theme/get-theme) :inverted)
```

Comment:
This removes the shadow at the top which was asked by @Francesca-G
CC: @VolodLytvynenko 
https://github.com/status-im/status-mobile/pull/19459#issuecomment-2044480588

Reply:
Thank you @ibrkhalil for reviewing this PR.
Please can you check now, I moved this shadow to new container in background to avoid showing on top.

Reply:
I approved the PR, But I think @Francesca-G is going to have a better eye in terms of reviewing the shadow.

Reply:
thank you